core/arch/arm/plat-imx/platform_config.h

```diff
@@ -114,7 +112,150 @@
  * The physical address is 0x2020000, we mapped it to 0x4020000,
  * see DEVICE0_VA_BASE and DEVICE0_PA_BASE
  */
-#define CONSOLE_UART_BASE	(0x4020000)
+#define CONSOLE_UART_BASE		(0x4020000)
 #define CONSOLE_UART_PA_BASE	(UART0_BASE)
 
+/* For i.MX6Q SABRE Lite EVK board */
+
+#elif defined(PLATFORM_FLAVOR_mx6qslevk)
+
+#define SCU_BASE                0x00a00000
+#define PL310_BASE              0x00a02000
+#define SRC_BASE				0x020D8000
+#define SRC_SCR					0x000
+#define SRC_GPR1				0x020
+#define SRC_SCR_ENABLE_OFFSET 	22
+#define GIC_BASE			    0x00a00000
+#define GICC_OFFSET			    0x100
+#define GICD_OFFSET			    0x1000
+#define GIC_CPU_BASE			(GIC_BASE + GICC_OFFSET)
+#define GIC_DIST_BASE			(GIC_BASE + GICD_OFFSET)
+#define UART0_BASE			    0x02028000
+#define UART1_BASE			    0x021E8000
+#define CSU_CSL_START			0x021c0000
+#define CSU_CSL_END				0x021c00a0
+#define CSU_CSL_5				0x021c0014
+#define CSU_CSL_16				0x021c0040
+#define CONSOLE_UART_BASE		UART1_BASE
+#define CONSOLE_UART_PA_BASE	UART1_BASE
```

Comment:
I think you should assign a new value for CONSOLE_UART_BASE. Virutal address <=64M is used by OPTEE kernel and OPTEE ta. 

Also check CSU and GIC address. I did not check the virual address, you need to make sure the virtual address > 0x4000000